redis/src/main/scala/zio/redis/ClusterExecutor.scala

```diff
@@ -18,18 +18,22 @@ package zio.redis
 
 import zio._
 import zio.redis.ClusterExecutor._
+import zio.redis.Input.StringInput
 import zio.redis.api.Cluster.AskingCommand
 import zio.redis.codec.StringUtf8Codec
 import zio.redis.options.Cluster._
 import zio.schema.codec.BinaryCodec
+import zio.stream.ZStream
 
 import java.io.IOException
 
 final case class ClusterExecutor(
   clusterConnectionRef: Ref.Synchronized[ClusterConnection],
   config: RedisClusterConfig,
-  scope: Scope.Closeable
-) extends RedisExecutor {
+  scope: Scope.Closeable,
+  codec: BinaryCodec
+) extends RedisExecutor
```

Comment:
Don't think that cluster executor should inherit RedisPubSub. 
I think it's more logic if it would be other way, there are would be some RedisPubSubClusterExecutor that is going to inherit ClusterExecuter, like SingleNodeRedisPubSubExecutor
And there is going to be some consistency.